src/Frame.h

```diff
@@ -183,8 +183,17 @@ namespace openshot
 		/// Display the wave form
 		void DisplayWaveform();
 
-		/// Get magnitude of range of samples (if channel is -1, return average of all channels for that sample)
-		float GetAudioSample(int channel, int sample, int magnitude_range);
+		/// Get highest value within a range of samples
+		/// 
+		/// @param channel Channel number to scan, or -1 for all channels
+		/// @param sample The starting sample for the range
+		/// @param sample_count The length of the sample range
+		/// @returns The maximum sample value within the range of samples on
+		///          the specified channel, or all channels if channel == -1
+		float GetPeakLevel(int channel, int sample, int sample_count);
```

Comment:
I started by renaming the method to `Frame::GetPeakLevel`. (Though I'm considering a change to `GetPeakAudioLevel` or `GetPeakSampleLevel`, now that I remember this is in `Frame` and not an audio class.) The new name better represents both its real purpose and its actual function. Its parameters are renamed, as mentioned, from `(channel, sample, magnitude_range)` to `(channel, sample, sample_count)`.

Reply:
(I did pull the trigger on renaming the method to `GetPeakSampleLevel`, which is why the code from this comment is now Outdated.)